core/prisma/exampleCommunitySeeds/arcadia.ts

```diff
@@ -693,7 +693,7 @@ export const seedArcadia = (communityId?: CommunitiesId) => {
 		},
 		{
 			randomSlug: false,
-			withApiToken: "xxxxxxxxxxxxxxxx.00000000-0000-0000-0000-000000000000",
+			withApiToken: "00000000-0000-0000-0000-000000000000.xxxxxxxxxxxxxxxx.",
```

Comment:
just want to confirm we do want the `.` at the end?

Reply:
whoops, we don't, but i think it's fine